DIRECTORY_REFACTORING_PLAN.md

```
@@ -0,0 +1,940 @@
+export function getEnsProfileUrl(ensName?: string | null): string | null {
+  return ensName ? `https://app.ens.domains/${ensName}` : null;
+}
+```
+
+---
+
+## 2. Component Extraction Plan
+
+### 2.1 New File Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)